content/en/docs/concepts/scheduling/configure-scheduler.md

```diff
@@ -0,0 +1,89 @@
+---
+title: How to Configure Your Kubernetes Scheduler
+reviewers:
+- wgliang
+- tengqm
+content_template: templates/concept
+weight: 30
+toc_hide: false
+---
+
+{{% capture overview %}}
+
+This page explains how to configure Kubernetes scheduler.
```

Comment:
This page seems more like a task description than a concept. Have I got that right?

Reply:
@sftim This is one of the series of the article so it named "how to configure ..." so not sure how many concepts we can put. But I can add more configuration options explanations in the future.